.gitlab-ci.yml

```diff
@@ -308,9 +308,9 @@ before_script:
 # stage: deploy-part1
   MOVED_TO_GROUP_VARS: "true"
 
-.coreos_vault_upgrade_variables: &coreos_vault_upgrade_variables
+.coreos_vault_variables: &coreos_vault_variables
 # stage: deploy-part1
-  UPGRADE_TEST: "basic"
```

Comment:
upgrade?

Reply:
first of all,need to deploy 1.11.3 cluster. then upgrade 1.12.1.
and kubeadm 1.11.3 supoort Final docker version is 17.03.  but coreos stable docker version is 18.06. deploy cluster will failed.

Reply:
we must test upgrades. doesn't have to be done on kubernetes.

Reply:
ye， We have already tested the deployment of the deployment by gce_centos-weave-kubeadm-sep test.

Reply:
OK  thx